process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
 | allowMarkup | Boolean | *Optional.* Allow recipients to make changes to your documents by covering up existing text and replacing it with new text (i.e. markup). Recipients can decide to use a special markup text field which they can place anywhere on the document. It can be scaled and optionally filled in. All changes must be reviewed and approved by all signers. |
+| file | File | *Required* File to be signed. If multiple files are received, only the first one will be processed. |
 
-> **Note**: `underscoreMetadata` can be set to `true` and the `targetFileMetadata` input can still use `:` with the connector successfully executing the action. If `underscoreMetadata` is set to `false` and `targetFileMetadata` uses `_` then the connector will fail to execute the action.
+The output parameters from `SEND_FOR_SIGNATURE` are:
+
+| Parameter | Type | Description |
+|--- | --- | --- |
+| envelopeId | String | Envelope ID of the document. |
+| status | String | Status of the envelope. |
+| URI | String | URI related to the envelope. |
+| docusignOutput | JSON | DocuSign output after sending the document for signature. |
 
-An example of the `targetFileMetadata` that can be stored with the document is:
+The input parameters for `DOWNLOAD_DOCUMENT` are:
 
-```json
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| outputFileName | String | *Optional.* The name of the file that will be created, for example `invoice.pdf`. |
+| envelopeId | yes | *Required* UUID related to the envelope from the DocuSign API. |
+| targetFolder | Folder | *Requires one.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type folder to store the signed document in. |
+| targetFolderId | String | *Optional.* The target folder ID to save the document in Content Services. |
+| targetFolderPath | String | *Requires one.* The location path or relative path of the folder to store the signed document in Content Services. For example, a location path: `/app:company_home/app:user_homes/cm:hruser` or a relative path: `/User Homes/hruser`.  |
+| targetFileType | String | *Optional.* The type to set for the signed file, for example `fin:invoice`. |
+| targetFileMetadata | content-metadata | *Optional.* Metadata assigned to the signed document in Content Services. |
+| underscoreMetadata | boolean | *Optional.* If set to `true` the received prefixed properties names contain underscores (_) instead of colons (:), for separating the namespace prefix from the property name. |
+
+The output parameters from `DOWNLOAD_DOCUMENT` are:
+
+| Property | Type | Description |
+|--- | --- |
+| file | file | The file stored in Content Services of the document. |
+
+The following is an example of the POST body for the Activiti REST API `http://{{domain}}/{{applicationName}}-rb/v1/process-instances` endpoint:
```

Comment:
I think this is more technical. Is there any kind of documentation like this in other sections? I can't see something similar in the rest of the connectors, for example.

Reply:
Removed the section and code.